src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -395,6 +396,14 @@ public function getAccessibleTabsAction(
         );
     }
 
+    #[AdminSecurity("is_granted('read', request.get('_legacy_controller'))")]
+    public function generatePasswordAction(): JsonResponse
+    {
+        return $this->json([
+            'password' => (new PasswordGenerator(new OpenSSL()))->generatePassword(16, PasswordGenerator::PASSWORDGEN_FLAG_ALPHANUMERIC),
```

Comment:
I see this is the same logic that
https://github.com/PrestaShop/PrestaShop/blob/1bb1e448e96744f1297666d224f0f5c8a4f621cf/src/PrestaShopBundle/Controller/Admin/LoginController.php#L240

so it makes sense 👍